### PR TITLE
:sparkles: Remove storybook dependency on Gulp

### PR DESCRIPTION
**Pull request by belen-albeza**
Closes https://tree.taiga.io/project/penpot/task/7845

To try this out:

1. ⚠️ **Stop the jobs of watching the assets and shadow cljs** (these are in tabs `0` and `1` of the tmux session in the dev env)
2. Create a new tab (`Ctrl + b c`) and run the storybook related commands there.

To do a build (to then been copied somewhere else or served from an HTTP server):

```zsh
yarn run storybook:build
```

To start watch mode (use this while developing components):

```zsh
yarn run storybook:watch
```

### CODE REVIEW

**Review by EvaMarco — APPROVED** [review]

**Review by Xaviju — COMMENTED** [review]
When the build task is launched, a new `frontend/storybook-static` folder appears as untracked in git. Shouldn't it be part of the `.gitignore` file?

**Comment by belen-albeza** [comment]
> When the build task is launched, a new `frontend/storybook-static` folder appears as untracked in git. Shouldn't it be part of the `.gitignore` file?

It is O_o https://github.com/penpot/penpot/pull/4607/commits/281a8748350ef7e0d6394fa26457b8c30f59913a#diff-bc37d034bad564583790a46f19d807abfe519c5671395fd494d8cce506c42947R60

Does running `git status` still show you that dir?

File: frontend/package.json [hunk 1]
```diff
@@ -32,9 +32,9 @@
     "watch": "node ./scripts/watch.js",
     "e2e:server": "node ./scripts/e2e-server.js",
     "e2e:test": "playwright test",
-    "storybook:compile": "gulp template:storybook && clojure -M:dev:shadow-cljs compile storybook",
-    "storybook:watch": "npm run storybook:compile && concurrently \"clojure -M:dev:shadow-cljs watch storybook\" \"storybook dev -p 6006\"",
-    "storybook:build": "npm run storybook:compile && storybook build"
+    "storybook:compile": "yarn run compile && clojure -M:dev:shadow-cljs compile storybook",
+    "storybook:watch": "yarn run storybook:compile && concurrently \"clojure -M:dev:shadow-cljs watch storybook\" \"storybook dev -p 6006\" \"yarn run watch\"" ,
```
Comment (Xaviju) [inline frontend/package.json]: Remove extra space at the end of the line

**Review by Xaviju — APPROVED** [review]
Seems that the `.gitignore` error was my mistake, it works perfectly.

There is only one minor change :)

**Pull request merged**